components/eam/bld/namelist_files/use_cases/1850_eam_CMIP6.xml

```diff
@@ -22,6 +22,11 @@
 <prescribed_volcaero_filetype>VOLC_CMIP6					       </prescribed_volcaero_filetype>
 <prescribed_volcaero_type    >CYCLICAL						       </prescribed_volcaero_type    >
 <prescribed_volcaero_cycle_yr>1                                                        </prescribed_volcaero_cycle_yr>
+<!-- Removing prescribed volcanic forcing for CLDERA -->
+<prescribed_volcaero_filetype cldera_strat_volc="1">VOLC_MIXING_RATIO                            </prescribed_volcaero_filetype>
+<prescribed_volcaero_datapath cldera_strat_volc="1">atm/cldera/emissions/volc/zero-presc-volc-mmr</prescribed_volcaero_datapath>
+<prescribed_volcaero_file cldera_strat_volc="1">BMW_volcanic_1850-3009_all_zero.nc               </prescribed_volcaero_file>
```

Comment:
Should this file also be a relative path within inputdata root, or does it exist at the top of inputdata root as implied here?

Reply:
The setup in the use_cases is to define an input datapath and the input file separately. So it is ok as it stands.